modules/trusted-firmware-m/CMakeLists.txt

```diff
@@ -32,8 +32,12 @@ set(TFM_CRYPTO_MODULES
 if (CONFIG_BUILD_WITH_TFM)
   if (CONFIG_TFM_IPC)
     list(APPEND TFM_CMAKE_ARGS -DTFM_PSA_API=ON)
```

Comment:
When they changed this they made TFM_PSA_API an internal variable, so this should be removed I think, according to how they document their options.
https://github.com/zephyrproject-rtos/trusted-firmware-m/commit/23a583c3e4b62b13c51eecb0e84f4c312af51ab4#diff-683a66548742d1b9880d0f96ed9d3d425064064320616f67d977d3c917e4ebb6L19